content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -146,68 +177,80 @@ most cases, you should follow the
-explore the SDK to see all the other input optionally supplied to this method.
+network. The `mplTokenMetadata` plugin we added earlier provides the required
+helpers to create an NFT or any other token with minimal configuration. The
+helper `createNft` method will create the mint account, token account, metadata
+account, and master edition account for you. The data provided to this method
+will represent the onchain portion of the NFT metadata. You can explore the SDK
+to see all the other input optionally supplied to this method.
 
 ```typescript
-const { nft } = await metaplex.nfts().create(
-  {
-    uri: uri,
-    name: "My NFT",
-    sellerFeeBasisPoints: 0,
-  },
-  { commitment: "finalized" },
-);
+const { signature, result } = await createNft(umi, {
+  mint,
+  name: "My NFT",
+  uri,
+  updateAuthority: umi.identity.publicKey,
+  sellerFeeBasisPoints: percentAmount(0),
+}).sendAndConfirm(umi, { send: { commitment: "finalized" } });
 ```
 
-This method returns an object containing information about the newly created
-NFT. By default, the SDK sets the `isMutable` property to true, allowing for
-updates to be made to the NFT's metadata. However, you can choose to set
-`isMutable` to false, making the NFT's metadata immutable.
+The `sendAndConfirm` method is what takes care of signing our transaction and
+sending it. It also provides other options to set pre-flight checks and our
+desired commitment for the transaction, which defaults to `confirmed` if not
+provided.
+
+This method returns an object containing the transaction signature and a result.
+The result object contains the outcome of our transaction. If successful, the
+`err` inside this will be set to null otherwise it'll contain the error for the
+failed transaction.
+
+By default, the SDK sets the `isMutable` property to true, allowing for updates
+to be made to the NFT's metadata. However, you can choose to set `isMutable` to
+false, making the NFT's metadata immutable.
 
 #### Update the NFT
 
-If you've left `isMutable` as true, you may update your NFT's metadata. The
-SDK's `update` method allows you to update both the onchain and offchain
+If you've left `isMutable` as true, you may update your NFT's metadata.
+
+The SDK's `updateV1` method allows you to update both the onchain and offchain
 portions of the NFT's metadata. To update the offchain metadata, you'll need to
 repeat the steps of uploading a new image and metadata URI (as outlined in the
 previous steps), then provide the new metadata URI to this method. This will
 change the URI that the onchain metadata points to, effectively updating the
 offchain metadata as well.
 
 ```typescript
-const nft = await metaplex.nfts().findByMint({ mintAddress });
-
-const { response } = await metaplex.nfts().update(
-  {
-    nftOrSft: nft,
+const nft = await fetchMetadataFromSeeds(umi, { mintAddress });
+
+await updateV1(umi, {
+  mint,
+  authority: umi.identity,
+  data: {
+    ...nft,
+    sellerFeeBasisPoints: 100, // 1%
```

Comment:
I don't think it's worth setting `sellerFeeBasisPoints` - they're not enforcable, at least one marketplace decided to sell NFTs cheaper rather than pay artist royalties. Transfer hooks or pNFTs are a better solution for that problem.

Reply:
Umi has it configured as a non-nullable field. We have to include it.

Reply:
Sorry , I should clarify: just set it to zero.